### ISSUE TITLE
Remove and replying an specific message

### ISSUE THREAD

**Issue by misisol** [bigbluebutton/bigbluebutton#18973]
**Is your feature request related to a problem? Please describe.**
Sometimes In some classes attendees sends something inappropriate in messages, and the moderator has to clear all the messages to get rid of that specific message.

**Describe the solution you'd like**
That would be nice for moderators to be able to remove and specific message in messages section, and also to be able to reply a message to.

**Comment by ffdixon** [comment]
Agreed -- it's definitely something we want to implement in a future version.

**Comment by antobinary** [comment]
Implemented as part of BBB 3.0.
